addon/models/guid-referent.js

```diff
@@ -0,0 +1,6 @@
+import OsfModel from './osf-model';
+
+/**
+ * Used only to support the polymorphic referent relationship on the Guid model
+ **/
+export default OsfModel.extend({});
```

Comment:
Hmm. This actually points out some holes in the v2 implementation... looking into this some more.